src/schema.graphql

```diff
@@ -152,3 +154,39 @@ input ModifyOrganizationInput {
   acronymEn: String
   acronymFr: String
 }
+
+type Approval {
+    id: ID!
+    gcIDApprover: Profile!
+    gcIDSubmitter: Profile
+    requestedChange: RequestedChange!
+    createdOn: String!
+    actionedOn: String
+    deniedComment: String
+    status: Status!
+    changeType: ChangeType!
+}
+
+type RequestedChange {
+    id: ID!
```

Comment:
Needed to add a RequestedChange id field becuase gcID is not a unique field to RequestedChange.